graylog2-server/src/test/java/org/graylog/testing/mongodb/MongoDBFixtureImporter.java

```diff
@@ -102,7 +102,7 @@ void importResources(MongoDatabase database) {
     }
 
     private static URL toResource(final String resourceName, final Class<?> contextClass) {
-        if (Paths.get(resourceName).isAbsolute()) {
+        if (Paths.get(resourceName).isAbsolute() || (Paths.get(resourceName).getNameCount() > 1)) {
```

Comment:
Previous to the change https://github.com/Graylog2/graylog2-server/commit/8b3e73e20d39777b376d33c7a76f51b5e0dfc925#diff-6474ec601eb016132c8b75872b017319R105
It was posible to write relative paths from resources root dir in the form of 

> `org/graylog/.../...`

this change allows it again.